verilog/analysis/verilog_linter.cc

```diff
@@ -299,8 +301,13 @@ absl::Status VerilogLintTextStructure(std::ostream* stream,
     VLOG(1) << "Lint Violations (" << total_violations << "): " << std::endl;
     // Output results to stream using formatter.
     verible::LintStatusFormatter formatter(contents);
-    formatter.FormatLintRuleStatuses(stream, linter_statuses, text_base,
-                                     filename);
+    if (!show_context) {
+      formatter.FormatLintRuleStatuses(stream, linter_statuses, text_base,
```

Comment:
So here, things would look like
`formatter.FormatLintRuleStatuses(stream, ..., show_context ? text_structure.Lines() : {});`

Reply:
Did you consider my suggestion to use the ?: operator for more compact and readable expression expression here ?

Reply:
Yes, but as far as I know the initializer_list can't be used inside the ternary operator. At least my compiler doesn't like such construction.
Am I right?

Reply:
I see. Ok, then this is probably best.